apps/www/src/lib/registry/default/ui/textarea/Textarea.vue

```diff
@@ -1,22 +1,14 @@
 <script setup lang="ts">
 import type { HTMLAttributes } from 'vue'
 import { cn } from '@/lib/utils'
-import { useVModel } from '@vueuse/core'
 
 const props = defineProps<{
   class?: HTMLAttributes['class']
   defaultValue?: string | number
-  modelValue?: string | number
 }>()
 
-const emits = defineEmits<{
-  (e: 'update:modelValue', payload: string | number): void
-}>()
-
-const modelValue = useVModel(props, 'modelValue', emits, {
-  passive: true,
-  defaultValue: props.defaultValue,
-})
+const modelValue = defineModel<string | number>()
+modelValue.value = props.defaultValue
```

Comment:
```suggestion
const modelValue = defineModel<string | number>({
  default: props.defaultValue
})
```

Reply:
It cannot be done like that, already tried it :grin: 
"[@vue/compiler-sfc] `defineModel()` in <script setup> cannot reference locally declared variables because it will be hoisted outside of the setup() function."